src/Command/Autoscaling/AutoscalingSettingsGetCommand.php

```diff
@@ -37,7 +37,7 @@ protected function execute(InputInterface $input, OutputInterface $output)
     {
         $this->validateInput($input);
         if (!$this->api()->supportsAutoscaling($this->getSelectedProject())) {
-            $this->stdErr->writeln(sprintf('The autoscaling API is not enabled for the project %s.', $this->api()->getProjectLabel($this->getSelectedProject(), 'comment')));
+            $this->stdErr->writeln(sprintf('The autoscaling feature is not enabled for the project %s.', $this->api()->getProjectLabel($this->getSelectedProject(), 'comment')));
```

Comment:
I thought "API" was clearer... to describe the potential for autoscaling being available, not that autoscaling is itself enabled.

Reply:
I would support "capability"... but at the moment, all the resources commands use "API": "The flexible resources API is not enabled...".